docs/reference/storage-reference.md

```diff
@@ -1,5 +1,109 @@
+| `search`           | **str**<br/>Search for assets that contain the provided search query in their name, title, or order ID.                                                                                                                                                                                                                                                                                                                                     |
+| `limit`            | **int**<br/>The number of assets on a result page.                                                                                                                                                                                                                                                                                                                                                                                          |
+| `sortby`           | **str**<br/>The property to sort by. The allowed values are:<br/><ul><li>`"id"`</li><li>`"workspaceId"`</li><li>`"accountId"`</li><li>`"createdAt"`</li><li>`"updatedAt"`</li><li>`"name"`</li><li>`"size"`</li><li>`"contentType"`</li><li>`"geospatialMetadataExtractionStatus"`</li><li>`"productId"`</li><li>`"orderId"`</li><li>`"producerName"`</li><li>`"collectionName"`</li><li>`"source"`</li>The default value is `"createdAt"`. |
+| `descending`       | **bool**<br/>Determines the sorting order of the returned assets.<br/><ul><li>`True`: descending</li><li>`False`: ascending</li></ul>The default value is `True`.                                                                                                                                                                                                                                                                           |
+| `return_json`      | **bool**<br/>Determines how to return the assets.<br/><ul><li>`True`: returns a JSON dictionary</li><li>`False`: returns a list of asset objects.</li></ul>The default value is `False`.                                                                                                                                                                                                                                                    |
+
+<h5> Example </h5>
+
+```python
+storage.get_assets(
+    created_after="2021-01-01",
+    created_before="2023-01-01",
+    workspace_id="68567134-27ad-7bd7-4b65-d61adb11fc78",
+    collection_names=["phr"],
+    producer_names=["airbus"],
+    tags=["optical", "us"],
+    sources=["ARCHIVE"],
+    search=["NY Central Park"],
+    limit=5,
+    sortby="productId",
+    return_json=True,
+)
+```
+
+## Orders
+
+### get_orders()
```

Comment:
Let's also support [this release](https://github.com/up42/up42-py/pull/465/files#diff-f9cb3611528e0a7c83ebe23981dd51766dab5af53b9d6396e714552008d75f41R314) here:
- New arguments: 
  - `workspace_orders: bool = True`
  - `status: Optional[List[str]] = None`
  - `name: Optional[str] = None`
- Status values: 
  - ` "CREATED",
            "BEING_PLACED",
            "PLACED",
            "PLACEMENT_FAILED",
            "DELIVERY_INITIALIZATION_FAILED",
            "BEING_FULFILLED",
            "DOWNLOAD_FAILED",
            "DOWNLOADED",
            "FULFILLED",
            "FAILED_PERMANENTLY"` (+ a link to the [description](https://docs.up42.com/developers/api-tasking/tasking-monitor#order-statuses))